source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -5,4 +5,140 @@ Access Data From a Cursor
 .. default-domain:: mongodb
 
 Read operations that return multiple documents do not immediately return
-all values matching the query.
+all values matching the query. Because a query can potentially match
+large sets of documents, these operations rely upon a
+:java-docs:`FindIterable
+<apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html>`.
+
+A ``FindIterable`` iterates through all the documents in your collection
+to find documents that match your search criteria. It also allows you to
+further specify which documents to see by setting paramaters though methods. 
+
+This page uses an initiating method, ``find()`` to show how to access
+data from a ``FindIterable``. 
+
+Terminal Methods
+----------------
+
+Execute database operations on a ``FindIterable`` using terminal
+methods. Terminal methods return a single value after iterating thorugh
+all the documents in a collection.
```

Comment:
This seems inaccurate, as `first` does not iterate through all the documents, just until it finds a match to the filter.